docs/topics/api/reviewers.rst

```diff
@@ -104,8 +104,11 @@ This endpoint allows you to browse through the contents of an Add-on version.
     :>json string validation_url_json: The url to the addons-linter validation report, rendered as JSON.
     :>json string validation_url: The url to the addons-linter validation report, rendered as HTML.
     :>json boolean has_been_validated: ``True`` if the version has been validated through addons-linter.
+    :>json object addon: A simplified :ref:`add-on <addon-detail-object>` object that contains only a few properties: ``id``, ``name``, ``icon_url`` and ``slug``.
     :>json object file: The file attached to this version. See :ref:`version detail -> files[] <version-detail-object>` for more details.
     :>json string file.content: Raw content of the requested file.
+    :>json string file.selected_file: The selected file, either from the ``file`` parameter or the default (``manifest.json``, ``package.json`` or ``install.rdf``).
```

Comment:
what's `package.json` used for?

Reply:
Also, `:param file:` above says it can be the xml file for search engine plugins too.

Reply:
it's just in there for historic reasons, as long as we don't hard-delete legacy add-ons we'll be able to review them and see them through the api. That'll all go away once stuff get's  hard deleted, I added it for completeness.

Reply:
Updated in 45fe5fdffb